osmaxx/excerptexport/models/excerpt.py

```diff
@@ -21,18 +26,35 @@ class Excerpt(models.Model):
     excerpt_type = models.CharField(max_length=40, choices=EXCERPT_TYPES, default=EXCERPT_TYPE_USER_DEFINED)
 
     COUNTRY_SIMPLIFICATION_TOLERANCE = 0.01
```

Comment:
While we're at it, rename this so that its unit (angular degrees, I suppose) is apparent.